Lib/test/test_generated_cases.py

```diff
@@ -466,6 +466,23 @@ def test_macro_instruction(self):
     """
         self.run_cases_test(input, output)
 
+    def test_unused_cache(self):
+        input = """
+        inst(OP, (unused/1 --)) {
+            body();
+        }
+    """
+        output = """
+        TARGET(OP) {
+            frame->instr_ptr = next_instr;
+            next_instr += 2;
+            INSTRUCTION_STATS(OP);
+            body();
+            DISPATCH();
+        }
+    """
+        self.run_cases_test(input, output)
```

Comment:
I'm not sure this test is covering everything that changed in this PR.

Reply:
Probably not, it is just a smoke test.
The real test is that everything continues to work properly.

Reply:
Actually, it would be nice if there was a test that adds a `/* Skip N cache entries */` comment where it wasn't before this PR, so we could see regressions around that. (I don't care about the singular/plural, but I care about it appearing at all.)